### PR TITLE
update mysql because tuple has changed

### LINKED ISSUE #575: 🐛 [Bug] Show all users with not activated account don't work anymore
## 🐛 Bugreport
In Old Backend the function to show all users which don't have activated here Gradido Account don't work anymore since our last release.

### PR DESCRIPTION
**Pull request by einhornimmond**
## 🍰 Pullrequest
Error occure because model::table::UserTuple has changed. Description was added, so I must update the custom mysql query for the search also.

### Issues
- fixes #575

### CODE REVIEW

File: login_server/src/cpp/JSONInterface/JsonGetUsers.cpp [hunk 1]
```diff
@@ -93,10 +93,10 @@ Poco::JSON::Object* JsonGetUsers::handle(Poco::Dynamic::Var params)
 	result->set("state", "success");
 
 	//Poco::JSON::Object jsonResultObject;
-	Poco::JSON::Array  jsonUsersArray;
+	Poco::JSON::Array::Ptr jsonUsersArray = new Poco::JSON::Array;
```
Comment (ulfgebhardt) [inline login_server/src/cpp/JSONInterface/JsonGetUsers.cpp:96]: Why change the type to Pointer? How is this related to the SQL problem?

**Review by ulfgebhardt — APPROVED** [review]

File: login_server/src/cpp/JSONInterface/JsonGetUsers.cpp (hunk 1 above)
Comment (einhornimmond) [reply login_server/src/cpp/JSONInterface/JsonGetUsers.cpp:96]: It is not related to SQL Problems, but I need as a type Pointer for testing.
